client/packages/common/src/ui/components/inputs/Filters/DateFilter.tsx

```diff
@@ -39,8 +41,14 @@ export const DateFilter: FC<{ filterDefinition: DateFilterDefinition }> = ({
         updateQuery({ [urlParameter]: { [range]: '' } });
         return;
       }
+
+      const date =
+        range === 'to' && displayAs === 'date' // if no time picker, set "TO" field to end of day
+          ? DateUtils.endOfDay(selection)
+          : selection;
```

Comment:
Cheeky little addition, so if querying a datetime field, e.g. createdDatetime, but its really only a date as far as user is concerned, make the "to" date of the range inclusive